### DISCUSSION TITLE
Device registration

### DISCUSSION THREAD

**Discussion by Talim** [ardevd/jlrpy#103] [Q&A]
Hi, first of all thank you for clean api documentation of JLR.

I would like to know if device registration has a lifetime ending or not. When access_token is consummed (and not using refresh_token mechanism at all), are we need to re-register a new deviceId (uuidv4) ? 

and if we use this one the refresh_token mechanism ? same question here

Thanks.

**Comment by ardevd** [comment]
Unfortunately, I honestly don't know. The device registration mechanism remains a bit of a mystery. Occasionally, I get the sense that there's a device number cap too where InControl will just decide to stop accepting new registrations, and I have no idea when/how the device registrations expire.

**Reply by Talim** [reply]
Thank you for your reply.
I appreciate responsiveness.

If i find anything about this after working on my implementation, I would write it here.
